assets/src/util/api.js

```diff
@@ -56,4 +56,38 @@ export default class Api {
       })
       .then((data) => data["hydra:member"]);
   }
+
+  static async fetchResource(apiEndpoint, resourceId) {
```

Comment:
This should be changed to use the new endpoint below "itkdev_booking.resources_endpoint_specific:"

I don't think the facilities should be added in this way. This should not come from the data, but be specified in the filter.